### ISSUE TITLE
Connecting web applications

### ISSUE THREAD

**Issue by jdebacker** [compute-tooling/compute-studio#404]
@hdoupe I enjoyed your demo day talk today and it rekindled a thought I've had about connecting the tax policy models on C/S.

Just as you passed reforms from one model to another in your demo day, could it be possible to create a set of "Run with..." buttons on the results pages of models?  E.g., on the Tax-Brain result page, one could do "Run with CCC" or "Run with OG-USA".  Clicking the button would bring one to the simulation page of CCC or OG-USA with the IIT+Payroll tax parameters used in the Tax-Brain run pre-filled.  One could then adjust other parameters (e.g., business tax parameters in CCC) and then run the simulation.  

It seems that the C/S API should make this somewhat easy and I'd like to have a go at it.  I'd just need some more instruction about how one could modify the results page to add a "Run with..." button.

**Comment by hdoupe** [comment]
Thanks @jdebacker. I like your idea to add a "Run with..." button. While there isn't an explicit `html` output type yet, you can use the `table` output to return HTML so that one of the outputs is a button.

One option is to send the user over to a page in the Stitch app that looks like:


-----
Create Simulation from PSLmodels/Tax-Brain # 1234
- [ ] Run with PSLmodels/Cost-of-Capital-Calculator
- [ ] Run with PSLmodels/Tax-Cruncher
- [ ] Run with PSLmodels/OG-USA

**[ RUN ]**


Is this similar to what you have in mind?

**Comment by jdebacker** [comment]
Yes - that idea of sending to the switch app would work.  Would that be easiest to maintain?
